index.js

```diff
@@ -1,7 +1,56 @@
-// Iteration 1: Names and Input
+let hacker1 ="Dalia";
+console.log(`The driver's name is ${hacker1}`)
 
+let hacker2 = 'Ben';
 
-// Iteration 2: Conditionals
+console.log(`The navigator name is ${hacker2}`);
 
+//necesito contar cuántas letras tiene cada variable
+let conteo1 = hacker1.length 
+let conteo2 = hacker2.length
 
-// Iteration 3: Loops
+console.log(`El hacker 1 tiene ${conteo1} y el hacker 2 tiene ${conteo2}`)
```

Comment:
at this moment we leave variables in spanish but you should avoid using it lets keep in english